.github/workflows/check.yml

```diff
@@ -0,0 +1,54 @@
+#
+#
+#  Copyright 2016 CUBRID Corporation
+# 
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+# 
+#       http://www.apache.org/licenses/LICENSE-2.0
+# 
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+# 
+name: github-checks
+on:
+  pull_request:
+    branches:
+      - develop
+      - release/eclipse_upgrade
+      - release/Daejeon-NIRS
```

Comment:
suggest (I'm not sure about wildcard *)
`- 'release/**'`

Reply:
When creating a new branch, it seems to be a good way to apply it without modifying the code. Thank you